test/common_device_type.py

```diff
@@ -277,6 +331,29 @@ def only_fn(slf, device, *args, **kwargs):
 
         return only_fn
 
+# Decorator that provides all available devices of the device type to the test
+# as a list of strings instead of providing a single device string.
+# Skips the test if the number of available devices of the variant's device
+# type is less than the 'devices_required' arg.
+class multidevice(object):
```

Comment:
I'm never going to remember if the devices available > devices required which number this things gives me.

Maybe `deviceCountAtLeast`?

Reply:
Sure.

Reply:
oh sorry, I meant the name of the decorator.  I'm never going to look at what the parameter is called...